Python/bytecodes.c

```diff
@@ -2424,6 +2424,9 @@ dummy_func(
                 opcode = executor->vm_data.opcode;
                 oparg = (oparg & ~255) | executor->vm_data.oparg;
                 next_instr = this_instr;
+                if (_PyOpcode_Caches[_PyOpcode_Deopt[opcode]]) {
+                    PAUSE_ADAPTIVE_COUNTER(this_instr[1].counter);
+                }
```

Comment:
How does it connect to the purpose of the PR? Or is it just a drive-by bugfix?

Reply:
It's unconnected.
This PR uncovered the bug, so I'll need to fix it before merging this. Might as well do it here.